js/interface.js

```diff
@@ -360,23 +366,23 @@
     var result = [];
     var hasActiveMenu = false;
 
+    // Grouping an array of one type of menu.
+    var packageTypeMenus = _.groupBy(menus, function(menu) {
```

Comment:
```suggestion
    var packageTypeMenus = _.groupBy(menus, 'package');
```

These are the same and applies to many lodash functions. See https://lodash.com/docs/4.17.15#filter

```js
_.groupBy(foo, 'bar')
_.groupBy(foo, (x) => {
  return x['bar'];
})
```

Reply:
@tonytlwu, I didn't know that, thanks, fixed